src/srf/boolean.cpp

```diff
@@ -181,16 +181,19 @@ SCurve SCurve::MakeCopySplitAgainst(SShell *agnstA, SShell *agnstB,
 }
 
 void SShell::CopyCurvesSplitAgainst(bool opA, SShell *agnst, SShell *into) {
-    SCurve *sc;
-    for(sc = curve.First(); sc; sc = curve.NextAfter(sc)) {
+#pragma omp parallel for
+    for(int i=0; i<curve.n; i++) {
```

Comment:
'for(auto sc = curve.First(); sc; sc = curve.NextAfter(sc)) {'
or
'for(SCurve sc = curve.First(); sc; sc = curve.NextAfter(sc)) {'
not possible?

Reply:
Hmm probably not I guess OpenMP won't be able to handle the iterators...

Reply:
@ruevs  yeah, I think OMP version 5.0 allows some iterators, but MSVC is still stuck at version 2.0.  BTW that really bothers me because OMP is a great way to do the most trivial kind of parallel processing.